canonical-websites/www.ubuntu.com
# Build the CI/CD webinar takeover

## Done
Build the CI/CD webinar takeover and the engage page.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Refresh the homepage until you see the new takeover
- Make sure it matches [the design](https://github.com/ubuntudesign/web-squad/issues/994#issuecomment-469625896)
- Make sure both the takeover and engage page matches [the copy doc](https://docs.google.com/document/d/1YfVwlrv41pSai0G0cukHt5ASkshpKOAPK_sZeS7ZmSk/edit#)


## Issue / Card
Fixes https://github.com/ubuntudesign/web-squad/issues/996
